DotNetCLI/Get-VisualStudioProjectNuGetPackages/Get-VisualStudioProjectNuGetPackages.psm1

```diff
@@ -1,4 +1,4 @@
-function Get-VisualStudioProjectNuGetPackages
+﻿function Get-VisualStudioProjectNuGetPackage
```

Comment:
Changing to singular is to conform to a convention/analyzer? I'm asking because making this plural has a meaning, i.e. you receive all the NuGet packages referenced by the project, so potentially you're not just getting a single entry of something.

Also, looking at the singular vs. plural name without context evokes a completely different idea (at least to me) about the script's behavior.

Reply:
The rule is:
> The cmdlet 'Get-VisualStudioProjectNuGetPackages' uses a plural noun. A singular noun should be used instead.
> PSScriptAnalyzer([PSUseSingularNouns](https://docs.microsoft.com/en-us/powershell/utility-modules/psscriptanalyzer/rules/usesingularnouns?view=ps-modules))

In this case you are returning a package list for a project. This is the same type of activity as returning a list of files for a directory with `Get-ChildItem` and that's singular as well. As far as I can recall all of built-in the `[array]`-returning cmdlets are singular.

Reply:
Understood, thanks!